src/rlx_mustache.erl

```diff
@@ -0,0 +1,230 @@
+%% The MIT License
+%%
+%% Copyright (c) 2009 Tom Preston-Werner <tom@mojombo.com>
```

Comment:
The when part will be tagged by the git timestamp, so just a note about where it came from would be a nice touch.